StarodubovSergiy/HW7/task3.py

```diff
@@ -0,0 +1,12 @@
+# Write a function that calculates the number of characters included in given string
+# input: "hello"
+# output: {"h":1, "e":1, "l":2,"o":1}
+
```

Comment:
look at this implementation:

def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result